### DISCUSSION TITLE
How to combine the different simulation into one model?

### DISCUSSION THREAD

**Discussion by Atekton** [Open-Systems-Pharmacology/Forum#1536] [Questions & Problems]
Hello,

I am a beginner in PBPK modeling and a user of PK-SIM. I would like to know if it is possible to combine models of the same compound but different individuals and doses into a single model in order to establish an optimized model.

Thank you for any suggestions.

Best regards.

**Comment by StephanSchaller** [comment]
Dear @Atekton , can you further elaborate on how this combination is optimizing the model? Or do you want to optimize (fit) parameters?

**Reply by Atekton** [reply]
Dear @StephanSchaller,
Thank you for your prompt reply.
Because I have experimental data from different individuals, after combination, I want the final model could fit the observed data well from different individuals.  Or does it means I need to optimize the parameter first? 

Best,
Atekton

**Reply by SCZ1174** [reply]
I think you can include all the simulations you want to participate in the optimization at the same time (with your observations) under the parameter identification module and then run the parameter optimization, which will take into account all the cases you included in the simulation.

**Reply by StephanSchaller** [reply]
Correct
